src/styles/base.css

```
@@ -0,0 +1,114 @@
+
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+* {
+  --text-color: rgb(var(--ctp-text));
+  --highlight-color: rgb(var(--ctp-rosewater));
+}
+
+body {
+}
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)